### PR TITLE
Add monthly update for April 2022

### PR DESCRIPTION
**Pull request by mbasmanova**
A complete list of commits for reference: https://gist.github.com/mbasmanova/5917f8a6c1a448b6d52bf0254da26305

### CODE REVIEW

File: velox/docs/monthly-updates/april-2022.rst [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+Debugging Experience
+====================
+
+* Add support for collecting runtime statistics for individual expressions.
+* Add printExprWithStats function to print expression tree annotated with runtime statistics. :pr:`1425`
+* Add number of splits and threads to printPlanWithStats output.
+* Add filters pushed down into table scan to printPlanWithStats output.
+* Add a mechanism to register a listener that is called upon task completion and receives task completion status, error details and runtime statistics. :pr:`1404`
+* Add a mechanism to register a listener that is called on destruction of ExprSet and receives runtime statistics. :pr:`1428`
+
```
Comment (kgpai) [inline velox/docs/monthly-updates/april-2022.rst:48]: Maybe have a section called bug fixes ? 
Some things to add there will be :
- Add support for complex types in simple function initialization
-  Fix lifetime in DriverCancelTest 
etc..

**Review by kgpai — COMMENTED** [review]
Thanks Masha.

File: velox/docs/monthly-updates/april-2022.rst (hunk 1 above)
Comment (mbasmanova) [reply velox/docs/monthly-updates/april-2022.rst:49]: I'd like to strike a good balance between listing everything and keeping the update readable.

Comment (mbasmanova) [reply velox/docs/monthly-updates/april-2022.rst:49]: One option for making bug fixes easily discoverable is to add bug-fix label to corresponding PRs.

**Review by kgpai — APPROVED** [review]

**Review by aditi-pandit — COMMENTED** [review]
Thanks !

**Pull request closed without merging**